tests/migration/register_system_in_textmode.pm

```diff
@@ -0,0 +1,42 @@
+# SLE15 migration tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Register the original system before migration
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+use testapi;
+use Utils::Architectures;
+use utils;
+use migration;
+use registration;
+use y2_base;
+
+sub run {
+    select_console 'root-console';
+
+    my ($self) = @_;
+
+    # Save VIDEOMODE and SCC_REGISTER vars
+    my $orig_videomode = get_var('VIDEOMODE', '');
+    my $orig_scc_register = get_var('SCC_REGISTER', '');
+
+    # Set vars to make yast_scc_registration work in ncures
+    set_var("VIDEOMODE", 'text');
+    set_var("SCC_REGISTER", 'console');
+    # remember we perform registration on pre-created HDD images
+    set_var('HDD_SP2ORLATER', 1);
```

Comment:
when is used this hack later?

Reply:
This is not used if registering system by SUSEConnect command, so I removed it.